src/services/package-manager/batch-import-export-service.ts

```diff
@@ -5,15 +5,17 @@ import {
     PackageExportTransport,
     PackageKeyAndVersionPair,
     PackageManifestTransport,
+    StudioPackageManifest,
     VariableManifestTransport
 } from "../../interfaces/package-export-transport";
 import {FileService, fileService} from "../file-service";
 import {studioService} from "../studio/studio.service";
 import {parse, stringify} from "../../util/yaml"
-import AdmZip = require("adm-zip");
-import * as fs from "fs";
 import * as FormData from "form-data";
 import {BatchExportImportConstants} from "../../interfaces/batch-export-import-constants";
+import {packageApi} from "../../api/package-api";
+import {Readable} from "stream";
+import AdmZip = require("adm-zip");
```

Comment:
q: Why is this using `require` instead of `import`?

Reply:
should be from auto-import, will fix it.